microservices/s3_to_redshift/s3_to_redshift.py

```diff
@@ -139,9 +138,25 @@ def copy_query(dbtable, batchfile, log):
 
         obj = client.get_object(Bucket=bucket, Key=object_summary.key)
         body = obj['Body']
+        csv_string = ''
+        if 'global_regex' in data:
+            body_stringified = body.read()
+            for line in body_stringified.splitlines():
+                for exp in data['global_regex']['regex']:
```

Comment:
A suggestion to control for line-lengths and support readability with something like:

```
inline_pattern = data['global_regex']['string_replace']['match']
inline_replace = data['global_regex']['string_replace']['replace']
for line in body_stringified.splitlines():
    string = line.replace(inline_pattern, inline_replace))
    for exp in data['global_regex']['regex']:
        parsed_line, matched = re.subn(
            exp['match'], exp['replace'],string)
```

I think this will also help you to isolate for the condition of the config not settting an `inline_pattern`/`inline_replace`

Reply:
I have made the suggested changes to the code.